cmd/ci-operator-prowgen/main.go

```diff
@@ -140,7 +256,13 @@ func generateJobs(
 			imagesTest = true
 		}
 		test := testDescription{Name: element.As, Target: element.As}
-		presubmits[orgrepo] = append(presubmits[orgrepo], *generatePresubmitForTest(test, org, repo, branch))
+		var podSpec *kubeapi.PodSpec
+		if element.ClusterSpec.Template == "" {
```

Comment:
So I understand this needs a patch to `ci-operator` so that its `TestStepConfiguration` struct has a new `ClusterSpec` field? Do you have the necessary changes to `ci-operator` published somewhere?